### PR TITLE
Fix psalm-1 `Query::class`, `QueryInterface::class`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | https://github.com/yiisoft/active-record/pull/173

### CODE REVIEW

File: src/Query/Query.php [hunk 1]
```diff
@@ -627,7 +589,7 @@ public function addSelect($columns): self
      *
      * @return array
      */
-    protected function normalizeSelect($columns): array
+    protected function normalizeSelect(array|ExpressionInterface|string $columns): array
```
Comment (darkdef) [inline src/Query/Query.php:592]: array|string|ExpressionInterface vs array|ExpressionInterface|string
May be order by simple types (a-z), classes&interfaces (a-z), null?
And add this to codestyle?

File: src/Query/Query.php [hunk 2]
```diff
@@ -528,17 +488,18 @@ protected function cleanUpTableNames(array $tableNames): array
...
+            }
+
             if ($tableName instanceof Expression) {
-                if (!is_string($alias)) {
-                    throw new InvalidArgumentException(
-                        'To use Expression in from() method, pass it in array format with alias.'
-                    );
-                }
                 $cleanedUpTableNames[$this->ensureNameQuoted($alias)] = $tableName;
             } elseif ($tableName instanceof self) {
```
Comment (darkdef) [inline src/Query/Query.php:499]: Static? Class not final

File: src/Query/Query.php [hunk 3]
```diff
@@ -279,10 +281,10 @@ public function column(): array
      *
      * @throws Exception|InvalidConfigException|Throwable
      *
-     * @return false|int|string|null number of records. The result may be a string depending on the underlying database
+     * @return bool|int|string|null number of records. The result may be a string depending on the underlying database
      * engine and to support integer values higher than a 32bit PHP integer can handle.
      */
-    public function count(string $q = '*'): false|int|null|string
+    public function count(string $q = '*'): bool|int|null|string
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/Query/Query.php:288]: done

File: src/Query/Query.php [hunk 4]
```diff
@@ -299,9 +301,9 @@ public function count(string $q = '*'): false|int|null|string
      *
      * @throws Exception|InvalidConfigException|Throwable
      *
-     * @return mixed the sum of the specified column values.
+     * @return bool|int|string|null the sum of the specified column values.
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/Query/Query.php:305]: done.

File: src/Query/Query.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Query/Query.php:595]: The order of the classes is fixed automatically by style.ci.

File: src/Query/Query.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Query/Query.php]: done.

**Pull request merged**